rero_mef/theme/views.py

```
@@ -29,8 +77,549 @@
+                        elif idobj:
+                            id_strs.append({"text": str(idobj)})
+            # Compose the display: authorized_access_point | [source] | IDs
+            display = " ".join(parts)
+            if id_strs:
+                chips = []
+                for id_item in id_strs:
+                    text = id_item["text"]
+                    if id_item.get("linking"):
+                        text = f'<span style="color:red">{text}</span>'
+                    if "url" in id_item:
+                        chips.append(Markup(f'<a href="{id_item["url"]}">{text}</a>'))
```

Ruff (0.15.7)

[error] Unsafe use of `markupsafe.Markup` detected

(S704)